cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -343,13 +346,19 @@ def _get_volume_stats(self):
         backend_name = self.configuration.safe_get('volume_backend_name')
         if not backend_name:
             backend_name = self.__class__.__name__
+
+        max_over_subscription_ratio = self.configuration.safe_get(
+            'max_over_subscription_ratio')
         data = {'volume_backend_name': backend_name,
                 'vendor_name': 'VMware',
                 'driver_version': self.VERSION,
                 'storage_protocol': 'vmdk',
                 'reserved_percentage': self.configuration.reserved_percentage,
                 'total_capacity_gb': 'unknown',
-                'free_capacity_gb': 'unknown'}
+                'free_capacity_gb': 'unknown',
+                'thin_provisioning_support': True,
```

Comment:
Are these two always `True`? Would anyone want to configure these to `False`?

Reply:
I think for our deployment we always support both of these